scripts/eval/yamls/mpt_eval.yaml

```diff
@@ -1,6 +1,7 @@
 max_seq_len: 2048
 tokenizer_name: EleutherAI/gpt-neox-20b
 seed: 1
+precision: amp_bf16
```

Comment:
should we just use fp32 for all evals? cc @abhi-mosaic

Reply:
fp32 does not work with FSDP